types/AbstractBaseType.php

```diff
@@ -451,6 +451,17 @@ public function rawValue($value) {
     }
 
     /**
+     * This is called when a single string is needed to represent this Type's current
+     * value as a single (non-HTML) string. Eg. in a dropdown or in autocompletion.
+     *
+     * @param string $value
+     * @return string
+     */
+    public function displayValue($value) {
```

Comment:
Mh, I think `stringifyValue()` or `getValueString()` might be more speaking names for that function?